core/src/main/java/amino/run/compiler/PolicyStub.java

```diff
@@ -299,12 +300,13 @@ public String getMethodContent(MethodStub m, boolean isDMMethod) {
      * @param isDMMethod
      * @return Stub code for RPC call based on method type.
      */
-    public String getMethodRPCContent(boolean isDMMethod) {
-        if (isDMMethod == true) {
-            return "$__result = $__makeKernelDMRPC($__method, $__params);";
-        } else {
-            return "$__result = $__makeKernelRPC($__method, $__params);";
+    public String getMethodRPCContent(MethodStub m, boolean isDMMethod) {
```

Comment:
suggestion: Please write Javadoc for the newly added input parameter.

Reply:
Change made.